Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM centos:6
```

Comment:
I've moved it to a folder. I'm still testing it, please do not merge until further notice.